cylc/rose/__init__.py

```diff
@@ -73,39 +73,28 @@
 Special Variables
 -----------------
 
-The Cylc Rose plugin has specific logic for handling a small group of special
-environment / template variables in the ``rose-suite.conf`` file:
+The Cylc Rose plugin provides two environment/template variables
+to the Cylc scheduler:
 
 ``ROSE_ORIG_HOST``
-   The plugin provides the hostname of the computer where the plugin runs as
-   an evironment variable.
+   The plugin provides the hostname of the computer where the plugin runs.
```

Comment:
Would be clearer to relate this back to the command the user ran. Can't remember if that's the `cylc install` or `cylc play` command though...

```
   The plugin provides the hostname of the computer where the plugin runs. I.E. the one where `cylc install` is run.
```

Reply:
Both.

Reply:
@oliver-sanders  Have a look at my lastest update

Reply:
I think this would be clearer if we relate this back to the command they ran (because they don't know where or when the plugin runs, the "plugin" isn't really a user-facing concept in the way that `cylc install` and `cylc play` are).